driver_cleanup.py

```diff
@@ -68,7 +68,13 @@ def parseLine(self, line):
         else:
             self.__nextParam += 1
         if self.driverDateAndVersion:
-            date, version = self.driverDateAndVersion.split(None, 1)
+            try:
+            	date, version = self.driverDateAndVersion.split(None, 1)
+            	if not version:
```

Comment:
```.split(None, 1)``` cannot return empty ```version```. It would return a list of 1 element, so it would be a ```ValueError```.
I suggest rewriting this in a more clean way:

```python
try:
    date, version = self.driverDateAndVersion.split(None, 1)
except ValueError:
    date, version = self.driverDateAndVersion, '1'
```